test/t8_forest_incomplete/t8_gtest_empty_local_tree.cxx

```diff
@@ -76,21 +78,28 @@ class local_tree:public testing::TestWithParam <t8_eclass_t> {
 };
 /* *INDENT-ON* */
 
+/** This structure contains a bitset with all 
+ * local trees on all processes to be removed.
+ */
+struct t8_trees
```

Comment:
```suggestion
struct t8_trees_to_remove
```

There exist alreay a struct ```t8_tree_t``` which might be confusing in this testcase.

Reply:
Done